core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -429,7 +429,11 @@ void core_mmu_set_prtn(struct mmu_partition *prtn)
 
 	ttbr = virt_to_phys(prtn->l1_tables[0][get_core_pos()]);
 
+#ifdef ARM32
+	write_ttbr0(ttbr | ((paddr_t)prtn->asid << (TTBR_ASID_SHIFT - 32)));
```

Comment:
`TTBR_ASID_SHIFT` is defined as `48` for both arm32 and arm64. Why do you need to subtract 32 there?

Reply:
My fault. You're absolutely right. I mixed up 32b and 64b MMU.